mapstory/importers.py

```diff
@@ -54,11 +54,13 @@ def import_file(self, *args, **kwargs):
 
         for layer in configuration_options:
             feature_type = layer.get('featureType')
-            datastore = feature_type['store']['name']
+            store_name = feature_type['store']['name']
```

Comment:
For clarity, the term datastore was used in other sections of the code so I thought it would make sense to use it here as well.